packages/cli-repl/src/completer.ts

```diff
@@ -0,0 +1,76 @@
+import { types as shellTypes } from 'mongosh-shell-api';
+/**
+ * Return complete suggestions given currently typed line
+ *
+ * @param {any} output - The output.
+ *
+ * @returns {string} The output.
+ */
+function completer(line: string): any {
+  // keep initial line param intact to always return in return statement
+  // check for contents of line with:
+  const splitLine = line.split('.');
```

Comment:
Is line the whole input? I guess for now it really does not matter, and hopefully in general, but do we need to support autocomplete in case of 'var x = db.coll.find'?

Reply:
line would be the whole input, yea! and it should work with the `var x = db.coll.findA`. I've just added a test for this case so we can keep track of it.